front/app/containers/MainHeader/Components/NavbarContent/MobileNavbarContent/FullMobileNavMenu.tsx

```diff
@@ -158,30 +145,33 @@ const FullMobileNavMenu = ({
           <ContentContainer
             // Screen reader will add "navigation", so this will become
             // "Full mobile navigation"
-            // Needed because there's also a different nav (see MobileNavbar/index)
+            // Needed because there's also a different nav (see MobileNavbarContent/index)
             aria-label={formatMessage(messages.fullMobileNavigation)}
           >
-            <StyledTenantLogo />
+            <Box mb="16px">
+              <StyledTenantLogo flex="0" />
+            </Box>
             <MenuItems>
               {navbarItemPropsArray.map((navbarItemProps) => {
                 const { linkTo, onlyActiveOnIndex, navigationItemTitle } =
                   navbarItemProps;
-
                 return (
                   <FullMobileNavMenuItem
                     key={linkTo}
                     linkTo={linkTo}
                     onlyActiveOnIndex={onlyActiveOnIndex}
                     navigationItemTitle={localize(navigationItemTitle)}
                     onClick={handleOnMenuItemClick(linkTo)}
+                    scrollToTop
```

Comment:
Yeah I thought about this while working on the scrolling ticket, but I felt that not scrolling to the top would be the preferred default :thinking: Since then it leaves the default handling to the browser instead which is generally pretty good at predicting where it should scroll to.

For instance, every time we go back to a previous URL we ideally want it to put us exactly where we were when we navigated away from it. We also have a number of elements on pages which change the URL but where we wouldn't want to scroll up to the top when the URL changes. For me it just felt easier to keep track of if we made the scrollToTop behaviour the one we explicitly set :thinking: Otherwise we'd have to think of all the cases where we _don't_ want to scroll to the top and set a prop for that, and I kept missing edge cases when I tried this :sweat_smile: